traefik/templates/_podtemplate.tpl

```diff
@@ -272,6 +272,11 @@
           {{- if .Values.metrics.prometheus.manualRouting }}
           - "--metrics.prometheus.manualrouting=true"
           {{- end }}
+          {{- if .Values.metrics.prometheus.headerLabels }}
+          {{- range $label, $headerKey := .Values.metrics.prometheus.headerLabels }}
+          - "--metrics.prometheus.headerLabels.{{ $label }}={{ $headerKey }}"
```

Comment:
```suggestion
          - "--metrics.prometheus.headerlabels.{{ $label }}={{ $headerKey }}"
```
CLI is documented with lower case: https://doc.traefik.io/traefik/observability/metrics/prometheus/#headerlabels